lib/kwtool/crawler/keyword_results.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Kwtool.Crawler.KeywordResults do
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Account.Schemas.User
+  alias Kwtool.Crawler.Queries.KeywordResultQuery
+  alias Kwtool.Repo
+
+  def search(%User{id: user_id}, params) do
```

Comment:
Should we return empty list if params is empty? By adding function clause above this one

```elixir
def search(_, %{}), do: []
```

Reply:
yup, good idea, fixed in https://github.com/byhbt/kwtool/pull/49/commits/b661d4a02195ebae0bc8e488667b23058f3fa5f4